ssh_test.go

```diff
@@ -1,9 +1,9 @@
-package gitdb
+package gitdb_test
 
 import (
 	"testing"
 )
 
-func TestGenerateSSHKeyPair(t *testing.T){
+func TestGenerateSSHKeyPair(t *testing.T) {
```

Comment:
maybe remove this unless a test is planned to go here.

Reply:
yea there is a plan to write the test

Reply:
on second thought I think you are right, the ssh funcs are all unexported and can't be unit tested so will drop the test file. Cheers